azure/rs_azure_networking/lb_test_cat.rb

```diff
@@ -147,17 +146,17 @@
   $lb_ip = @lb_ip.address
 end
 
-define add_to_lb(@server1,@my_pub_lb) return @server1,@my_target_nic do
+define add_to_lb(@server,@my_pub_lb) return @server1,@my_target_nic do
```

Comment:
return @server1 needs to be @server

Reply:
also maybe you didn't rename server1

Reply:
Nope, I did to try to avoid confusion with the actual server1.